src/routes/write-exercise/components/__tests__/InteractionSection.spec.tsx

```diff
@@ -162,10 +163,10 @@ describe('InteractionSection', () => {
       false
     )
     const inputField = getByPlaceholderText(labels.exercises.write.insertAnswer)
-    fireEvent.changeText(inputField, 'Der zweipolige Phasenprüfer')
+    fireEvent.changeText(inputField, 'der kontaktlose Spannungsprüfer')
```

Comment:
Here no clear mock was called, which could explain why this test was still successful.